generators/utils.js

```diff
@@ -268,7 +268,6 @@ function replaceTranslation(body, generator) {
         }
     };
 
-    replaceRegex(/(\{translate\('([a-zA-Z0-9.\-_]+)'(, ?null, ?'.*')?\)\})/g, '""');
```

Comment:
@agaspardcilia on closer look I think this might still be needed for cases where we use 
`<element prop={translate('foo')} />`
SO lets leave it in if its not causing any issues